config/config.php

```diff
@@ -2,6 +2,7 @@
 //The main config file
 define('BASE_URL', '');
 define('ENV', 'dev');
+define('MODELDEBUG', 'false');
```

Comment:
Great idea. You need to define it like this. Do not enclose the false in quotes. That would turn the value to a string. Tested it and it works.
define('MODELDEBUG', false);